OWML.ModHelper.Menus/ModPromptButton.cs

```diff
@@ -21,7 +24,7 @@ public ScreenPrompt Prompt
                 _hotkeyButton.SetPrompt(value);
                 if (_prompt.GetText() != DefaultTitle)
                 {
-                    Object.Destroy(Button.GetComponentInChildren<LocalizedText>());
+                    UnityEngine.Object.Destroy(Button.GetComponentInChildren<LocalizedText>());
```

Comment:
apparently `using System` and `Object` don't like eachother

Reply:
do `using Object = UnityEngine.Object;`, then you can do `Object.Destroy`